RELEASING.md

```diff
@@ -2,20 +2,17 @@
 
 We follow the `git flow` [release process](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
 
-- [ ] Pull the most recent versions of `develop` and `master`
-- If normal release:
-  - [ ] Branch off of `develop` and name the branch `release-X.X.X`
-- If hotfix/patch:
-  - [ ] Branch off of `master` and name the branch `hotfix-X`
+- [ ] Pull the most recent versions of `main`
+- [ ] Branch off of `main` and name the branch `release-X.X.X` or `hotfix-X`
 - [ ] Update version number within the repository
   - in `setup.py`
   - in Copyright Notice in `README`
   - in the `LICENSE`
 - [ ] Update classifiers in `setup.py`
 - [ ] Verify that all tests pass (`python setup.py test`)
-- [ ] Commit the changes, push to GitHub, and start a pull request into `master`
-- [ ] Approve PR and merge it into master, but do not delete it yet
```

Comment:
`main`

Reply:
Also below

Reply:
Those is the old version, it has been all changed to main?

Reply:
Yeah this is on the neg diff. The pos diff below is right

Reply:
Weird, my comments went on the old code, supposed to be on the new code. Just noticed a few places where main was in ticks and a few places it wasn't, just pushing for them all to be consistently `main`.

Reply:
My bad, I clicked on the wrong side of the line

Reply:
oh the ticks :+1: